src/bin/lua/basic.lua

```diff
@@ -426,3 +426,15 @@ function get_is_function(t)
 	end
 	return _is_functions[t] or search_base(t, _base_is_functions) or "tolua_isusertype"
 end
+
+function clean_to_function_default(t, def)
```

Comment:
This function name says nothing really abou what it does, and there is little documentation.